### DISCUSSION TITLE
Headscale seems broken since v0.23.0

### DISCUSSION THREAD

**Discussion by mx4k** [tteck/Proxmox#3809] [Help Needed] [answered]
Good time of the day. Headscale has had major code updates and the newest release doesn't work properly. Tried with clean install and with update. Can somebody confirm?

**Comment by tteck** [comment, marked as answer]
More than likely, it's due to a misconfigured `config.yaml`.

**Reply by mx4k** [reply]
The only thing I set is the server_url: to a local ip. But neither the webui nor connecting with tailscale client works. (On a clean insatall).

**Reply by mx4k** [reply]
You are right, shame on me. You need to edit (at least) the "listen_addr" line too.
